bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/RefreshingOAuth2CredentialsInterceptor.java

```diff
@@ -86,12 +85,12 @@ public HeaderCacheElement(IOException exception) {
 
     public CacheState getCacheState() {
       long now = System.currentTimeMillis();
-      if (now > expiresTimeMs) {
-        return CacheState.Expired;
-      } else if (now > staleTimeMs) {
+      if (now < staleTimeMs){
+        return CacheState.Good;
```

Comment:
we return CacheState.Good when  now =< staleTimeMs && now =< expiresTimeMs, what is the reason for changing the logic here?